bokehjs/gulp/tasks/generate_defaults.py

```diff
@@ -104,8 +104,5 @@ def output_defaults_module(filename, defaults):
 
 output_defaults_module(filename = os.path.join(dest_dir, 'common/defaults.coffee'),
                        defaults = all_json)
-output_defaults_module(filename = os.path.join(dest_dir, 'widget/defaults.coffee'),
+output_defaults_module(filename = os.path.join(dest_dir, 'models/widgets/defaults.coffee'),
```

Comment:
@bryevdv, the files you are referring to are actually "required" in `common/base`, so putting them outside source tree is cumbersome (or at least needs reconfiguration of browserify). One thing I want to put on the table, is that I'm going to auto-generate models' paths, so this renaming task may be a no-op, because I may be able to generate them on-the-fly when generating the final bundle.